packages/framework/src/Resources/views/Admin/Content/Country/list.html.twig

```diff
@@ -4,12 +4,13 @@
 
 {% block h1 %}{{ 'Countries overview'|trans }}{% endblock %}
 
+{% block btn %}
```

Comment:
I guess you haven't done this on other pages (`Settings >  Lists and nomenclatures >  Flags`, `Settings >  Lists and nomenclatures >  Parameters`, `Settings >  Lists and nomenclatures >  Status of orders`, `Pricing >  Promo codes`) because the use inline editing, right?

![image](https://user-images.githubusercontent.com/10008612/61798114-073f6c80-ae29-11e9-8462-769d2bce5f12.png)

Reply:
If I would be honest - when i found add button outside of block btn a tried to move it. But on some pages I didn´t find add button. Is there any posibilty to review & unify it?

Reply:
After discussion we will unify it later.